### PR TITLE
add all mercedes-benz vehicles to common.js #8

### PR DESCRIPTION
**Pull request by hasalassh**
add all mercedes-benz vehicles to common.js #8

### CODE REVIEW

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Seems that there are some missing models? For example the following models all go in `gs` but are missing. 
```
190E (6-cyl 2.6L & 4-cyl 16v)
C230 (1999-2007)
CLA250 (FWD)
```

File: common.js [hunk 1]
```diff
@@ -4668,6 +4668,267 @@ const allSoloCars = {
...
+    },
+    'E550 (non-AMG)': {
+      '2014': ['stu'],
+      '2015': ['stu'],
+      '2016': ['stu']
+    },
+    'CLK 320 & CLK 32 AMG': {
+      'all': ['asp']
+    },
+    'E36 AMG': {
```
[the remark this answers is not in the document]
Comment (hasalassh) [reply common.js:4874]: There is E36 AMG (2010-16) under A Street Prepared (ASP.

**Comment by hasalassh** [comment]
> Seems that there are some missing models? For example the following models all go in `gs` but are missing.
> 
> ```
> 190E (6-cyl 2.6L & 4-cyl 16v)
> C230 (1999-2007)
> CLA250 (FWD)
> ```

@Bjorn248 Those models are under MERCEDES. That is why I have missed those. I have only considered models under mercedes-benz. Do you want me to include all models under MERCEDES too here?

**Comment by Bjorn248** [comment]
Yes please! I believe they are the same company.

File: common.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bjorn248) [reply common.js:4874]: In that case it is a Typo in the PDF. We should correct it in this data set though.

[the remark this answers is not in the document]
Comment (hasalassh) [reply common.js:4874]: @Bjorn248 Ok. Thanks for the confirmation. I will modify it as E63 AMG...!

**Comment by hasalassh** [comment]
@Bjorn248 I have done the requested modifications...!

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**